cleanlab/filter.py

```diff
@@ -53,18 +54,18 @@
 
 
 def find_label_issues(
-    labels,
-    pred_probs,
+    labels: np.ndarray,
```

Comment:
Annotating `labels` in _this particular function_ opens a whole can of worms with other mypy-errors.
Using `cleanlab.typing.LabelLike` does not seem to resolve those errors at this time.

I recommend that we leave this argument unannotated for now so we can keep the type-checker happy and work on merging your other changes.

```suggestion
    labels,
```